### ISSUE TITLE
SER Training: Fine-tune SVM classifier model accuracy

### ISSUE THREAD

**Issue by andyAndyA** [andyAndyA/Speech-Emotion-Analytics#10]
I would like to: Fine-tune SVM classifier model accuracy

So that I can: Deliver an actual somewhat usable SER engine

**Comment by andyAndyA** [comment]
Done in [improve-model-accuracy](https://github.com/andyAndyA/Speech-Emotion-Analytics/tree/improve-model-accuracy) branch, with highest accuracy rates at 77% and 71% when predicting against a testing subset from the original training data. Can be improved much more, but will do as a demo deliverable for now.
